BUILD.md

```diff
@@ -67,90 +76,86 @@ sudo pip3 install conan==1.59 grpcio-tools==1.57.0
 ```bash
 mkdir ~/ydbwork && cd ~/ydbwork
 mkdir build
-
 ```
 
+## Install ccache
+
+1. Install `ccache` into `/usr/local/bin/` (We are using version `4.8.1`, you can use any version greater than `4.7`)
+    ```bash
+    (V=4.8.1; curl -L https://github.com/ccache/ccache/releases/download/v${V}/ccache-${V}-linux-x86_64.tar.xz | \
+     sudo tar -xJ -C /usr/local/bin/ --strip-components=1 --no-same-owner ccache-${V}-linux-x86_64/ccache)
+    ```
+
+2. Configure `ccache` to use remote storage of YDB build artifacts, to speed up first build time:
+    ```bash
+    ccache -o remote_storage="http://cachesrv.ydb.tech:8080|read-only|layout=bazel"
+    ccache -o sloppiness=locale 
+    ccache -o base_dir=~/ydbwork/
+    ```
+
+If you use a non-default work directory, correct the `base_dir` ccache option to match it.
+
 ## Clone the ydb repository.
 
 ```bash
 git clone https://github.com/ydb-platform/ydb.git
```

Comment:
So that GitHub authentication is not needed. Instructions on how to develop (make changes) will be in a separate document, and there will be git protocol there.